Dockerfile

```
@@ -56,14 +58,11 @@ RUN \
   -O /usr/local/bin/winetricks && \
   chmod +x /usr/local/bin/winetricks
 
-RUN adduser --disabled-password --gecos ''  user && \
+RUN adduser --disabled-password --uid ${ZWIFT_UID} --gecos ''  user && \
```

```suggestion
RUN adduser --disabled-password --gecos ''  user && \
```
You change the UID in the entrypoint, so no need to specify this in the Dockerfile afaics.